subworkflows/local/arg.nf

```diff
@@ -91,9 +92,16 @@ workflow ARG {
 
     // DeepARG prepare download
     if ( !params.arg_skip_deeparg && params.arg_deeparg_data ) {
-        ch_deeparg_db = Channel
-            .fromPath( params.arg_deeparg_data )
-            .first()
+
+        if ( file(params.arg_deeparg_data).getExtension() == "zip") {
```

Comment:
Why `.zip` suffix? The downloaded file is `tar.gz`, or did you observe `zip` somewhere?

Reply:
Zenodo is zip I think?

Reply:
DeepARG database is not on Zenodo yet :D That's was the problem (I'll upload it in `tar.gz` format).

Reply:
Oh wait, I take that back. I completely missed that they uploaded the files! I will try with that ZIP again...